runatlantis.io/docs/using-atlantis.md

```diff
@@ -167,6 +167,49 @@ atlantis import -d dir 'aws_instance.example["foo"]' i-1234567890abcdef0 -- -var
 ```
 If a flag is needed to be always appended, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
 
+---
+## atlantis state rm
+```bash
+atlantis state [options] rm ADDRESS... -- [terraform import flags]
```

Comment:
Does the `...` mean that we can provide multiple items to remove in a single line ?

```shell
atlantis state rm 'random_string.foo' 'random_string.bar'
```

Reply:
Nice! Lets add state rm of multiple addresses as an example so users will know about it

Reply:
I see it here

https://github.com/runatlantis/atlantis/pull/2880/files#r1058008284